RGPosadas/LitterRally
# US-8.1: Add Friends

closes #273

Note:
- Some e2e tests were failing. I fixed them all (aside from US-2.8 which is sometimes flaky for me). they should all pass if you run them from a freshly seeded DB BEFORE 11:45pm

How to test
- Go to your profile page
- Go to "Add Friends"
- Make sure searching for strangers, accepting friend requests, and canceling pending requests works.
